### PR TITLE
Keep additional events - rename Task to Property

### PR DESCRIPTION
**Pull request by regisoc**
Change the `TaskName` and `TaskValue` to `PropertyName` and `PropertyValue` for additional event integration to match BIDS naming.

Linked to [LORIS 8432](https://github.com/aces/Loris/pull/8432)

### CODE REVIEW

**Review by jeffersoncasimir — APPROVED** [review]

File: python/lib/physiological.py [hunk 1]
```diff
@@ -704,8 +704,8 @@ def insert_event_file(self, event_data, event_file, physiological_file_id,
                 # table cols
                 add_event_fields = (
                     'PhysiologicalTaskEventID',
```
Comment (christinerogers) [inline python/lib/physiological.py:706]: Could `PhysiologicalTaskEventID` be changed too?

**Review by christinerogers — CHANGES_REQUESTED** [review]
see my comments on #8432 as well in Loris repo

**Comment by cmadjar** [comment]
Blocked by https://github.com/aces/Loris/pull/8432 which need to be merged at the same time as this PR on main.

**Review by cmadjar — CHANGES_REQUESTED** [review]
@regisoc isn't this supposed to go on main since the SQL changes go to main on the LORIS side?

**Comment by regisoc** [comment]
Yes, but we agreed on the `try/catch` bloc in `LORIS-MRI` that will make the routing depending on the code version in `LORIS`: `24.1` does not have the SQL change on LORIS, but `main` has. I was told to push this on `24.1` but I can change it if that is not convenient anymore.

**Comment by cmadjar** [comment]
@regisoc right, so if SQL is the main version, the code changed here will be run, otherwise it will still run the old code. Sounds good then

**Comment by cmadjar** [comment]
@christinerogers is that OK with you to get merged? I see you requested changes but the sentence says `looks ok to me...`. If all good, could you please edit your review and approve the PR? Thanks!

**Review by christinerogers — APPROVED** [review]

**Pull request merged**